Source/Scene/Cesium3DTileset.js

```diff
@@ -332,51 +336,11 @@ define([
         when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
     }
 
-    function selectTileWithTilesetContent(tiles3D, selectedTiles, tile, fullyVisible, frameState, additiveRefinment) {
-        // 1) If its children are not loaded, load the subtree it points to and then select its root child
-        // 2) If its children are already loaded, select its (root) child since the geometric error of it is
-        //    same as this tile's
-
-        // If the subtree has already been added and child
-        // content requested, select the child (= the root) and continue
-        if ((tile.isReady()) &&
-            (tile.numberOfChildrenWithoutContent === 0)) {
-            // A tiles.json must specify exactly one tile, ie a root
-            var root = tile.children[0];
-            if (root.hasTilesetContent) {
-                selectTileWithTilesetContent(tiles3D, selectedTiles, root, fullyVisible, frameState, additiveRefinment);
-            } else {
-                if (root.isContentUnloaded()) {
-                    requestContent(tiles3D, root);
-                } else if (root.isReady()){
-                    selectTile(selectedTiles, root, fullyVisible, frameState);
-                }
-            }
-            return;
-        } else if (!additiveRefinment) {
-            // Otherwise, for replacement refinment, select the parent tile to avoid
-            // showing an empty space while waiting for tile to load
-            if (defined(tile.parent)) {
-                selectTile(selectedTiles, tile.parent, fullyVisible, frameState);
-            }
-        }
-
-        // Request the tile's tileset if it's unloaded.
-        // In the case of tileset content tiles, their state must be explicitly set
-        if (tile.isContentUnloaded()) {
-            tile.content.state = Cesium3DTileContentState.LOADING;
-            var contentUrl = tile._header.content.url;
-            var tilesUrl = (new Uri(contentUrl).isAbsolute()) ? contentUrl : tiles3D._url + contentUrl;
-            loadTilesJson(tiles3D, tilesUrl, tile, function() {
-                tile.content.state = Cesium3DTileContentState.READY;
-            });
-        }
-    }
-
     function selectTile(selectedTiles, tile, fullyVisible, frameState) {
         // There may also be a tight box around just the tile's contents, e.g., for a city, we may be
         // zoomed into a neighborhood and can cull the skyscrapers in the root node.
-        if (tile.isReady() &&
+        var parentRefines = defined(tile.parent) ? tile.parent.isRefinable() : true;
```

Comment:
A better name might be `refinableParent`.

Can you also add a comment here that explains why this is checked, e.g., because we don't want to select a tile for replacement refinement unless all the tiles needed to replace the parent are ready.

Also, do we only need this check for replacement refinement?  Does this always pass for additive refinement?